src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+
+    }
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("---------------------").append("\n");
+        stringBuilder.append("Ball color: ").append(color).append("\n");
```

Comment:
You're using the default toString() method of the 'color' object. If 'color' is an enum, it's better to use 'color.name()' to ensure the proper string representation is used.

Reply:
deleted this information output